core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -331,6 +380,20 @@ public void finished(long requestSize, long responseSize, long latencyMs) {
       requestSizeSensor.record(requestSize);
       responseSizeSensor.record(responseSize);
       requestLatencySensor.record(latencyMs);
+
+      if (enableLatencySloSla) {
+        if (latencyMs < latencySloMs) {
+          responseLatencySloSlaSensor.get(LATENCY_SLO_SLA_SENSOR_NAMES[0]).record();
```

Comment:
Again, it would be more expressive if the sensor names had separate constants so that it was not necessary to refer back to the array to work out what the members actually are.

Reply:
Thanks for the comments! 
Yea, it does seem kind of awkward to look back and forth to the array. I followed the [previous people' approach](https://github.com/confluentinc/rest-utils/blob/master/core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java#L199-L225) and separated them because they have separate content and hopefully it's more readable.